js/packages/openinference-instrumentation-llama-index/src/utils.ts

```diff
@@ -56,14 +63,94 @@ function getExecContext(span: Span) {
   return execContext;
 }
 
+/**
+ * If execution results in an error, push the error to the span.
+ * @param span
+ * @param error
+ */
+function handleError(span: Span, error: Error | undefined) {
+  if (error) {
+    span.recordException(error);
+    span.setStatus({
+      code: SpanStatusCode.ERROR,
+      message: error.message,
+    });
+    span.end();
+  }
+}
+
+function documentAttributes(
```

Comment:
might rename this i like to be descriptive although it does become verbose (can look at examples in langchain instrumentation):

- getDocumentAttributes
- getDocumentAttributesFromRetrievalOutput
- formatDocumentAttributes